scripts/xbstrap

```diff
@@ -1774,6 +1858,23 @@ def do_list_pkgs(args):
 
 do_list_pkgs.parser = main_subparsers.add_parser('list-pkgs')
 
+def do_run_target(args):
+	args.all = False
+	args.update = False
+	args.reconfigure = False
+	args.rebuild = False
+	cfg = config_for_dir()
+	sel = select_pkgs(cfg, args)
+	plan = Plan(cfg)
+	update_pkg_sources(cfg, plan, args, sel)
+	reconfigure_and_rebuild_pkgs(plan, args, sel)
+	plan.wanted.update([(Action.RUN_PKG, RunPkgTarget(cfg, pkg, args.target[0])) for pkg in sel])
```

Comment:
Instead of creating the `RunPkgTarget` here, I think we should create it when initializing the `_run_targets` dict.

Reply:
Addressed in 12a0f8f.